docs/section4/4-6.md

```diff
@@ -0,0 +1,38 @@
+# Airside Systems – Rule 4-6
+
+**Rule ID:** 4-6
+**Rule Description:** HVAC fans shall remain on during unoccupied hours in systems primarily serving computer rooms.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules exception #3 for the proposed building  
+**Appendix G Section Reference:** None
+**Data Lookup:** None
+
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem 
+**Applicability Checks:** 
+
+1. Applies to hvac systems primarily serving computer rooms.
+
+**Function Call:** 
+
+1. get_zones_computer_rooms()
+
+**Manual Check:** None  
+ 
+**Rule Logic:**
+**Applicability Check 1:** 
+- Get list of zones that have computer rooms associated with them: `zones_serving_comp_rooms_list_p = get_zones_computer_rooms(P_RMR)`
+- For each zone in P_RMR: `for zone_p in P_RMR..Zone:`
+    - Check if the zone serves computer room(s), if yes then set zone serves computer room boolean variable to true: `if zone_p.id in zones_serving_comp_rooms_list_p: zone_primarily_serves_comp_room_check = TRUE`
+    - Check if zone serves computer room boolean variable is true, if so then carry on with rule logic: `if zone_primarily_serves_comp_room_check == TRUE:`
+        - For each terminal in zone_p: `for terminal_p in zone_p.terminals:`
+            - Get HVAC system serving terminal: `hvac_p = terminal_p.served_by_heating_ventilation_air_conditioning_systems`
+            - Add to list of heating_ventilation_air_conditioning_systems_p as the code loops through the terminal units: `heating_ventilation_air_conditioning_systems_list_p = heating_ventilation_air_conditioning_systems_list_p.append(heating_ventilation_air_conditioning_systems_p)`                    
+        - Add to master list of HVAC system that are applicable to this check after looping through each terminal: `applicable_hvac_systems_list_p = applicable_hvac_systems_list_p.append(heating_ventilation_air_conditioning_systems_list_p)`                    
+- Convert the list of applicable_hvac_systems_list_p to a set and the back to a list to eliminate duplicates after looping through all zones:`applicable_hvac_systems_list_p = list(set(applicable_hvac_systems_list_p))`                             
+- For each hvac_p in applicable_hvac_systems_list_p: `for each hvac_p in applicable_hvac_systems_list_p:`                         
```

Comment:
1. To simplify the RDS, should all the text above be replaced with path keyword to locate all HVAC systems serving zones included in the get_zones_computer_rooms(P_RMR) list. 
2. See also my comment on the function - I think it should provide a list of systems "primarily" serving computer rooms. Alternatively, this logic should be moved into the RDS itself instead of having a separate function.

Reply:
I kept the function for now because I believe I used on another RDS. I can consolidate later if that is okay.